contextvh/src/main/java/contextvh/actions/GetRelevantAreasBuild.java

```diff
@@ -26,6 +28,10 @@
  */
 public class GetRelevantAreasBuild implements RelevantAreasAction {
 
+	private final int DEFAULT_MAX_POLYS = 10;
```

Comment:
You should make these static or change the names, but statics would be the way to go.